go-backend/config/config.go

```diff
@@ -26,20 +29,39 @@ func Load(configFile string) {
 	viper.AutomaticEnv()
 }
 
+// AppName - returns the app name
 func AppName() string {
 	if appName == "" {
 		appName = ReadEnvString("APP_NAME")
 	}
 	return appName
 }
 
+// AppPort - returns application http port
 func AppPort() int {
 	if appPort == 0 {
 		appPort = ReadEnvInt("APP_PORT")
 	}
 	return appPort
 }
 
+// JwtKey - returns the JSON Web Token key
+func JwtKey() []byte {
```

Comment:
should be `JWTKey`
as a standard abbreviations should be all caps

Reply:
Upcased.